assets/services.json

```diff
@@ -0,0 +1,442 @@
+[
+  {
+    "id": "9gag",
+    "name": "9GAG",
+    "domain_rules": [
+      "||9cache.com^",
+      "||9gag.com^"
+    ],
+    "icon": "<svg xmlns=\"http://www.w3.org/2000/svg\" className=\"d-none\">\n<symbol id=\"service_9gag\" viewBox=\"0 0 50 50\" fill=\"currentColor\">\n<path d=\"M 44 14 C 44 13.644531 43.8125 13.316406 43.507813 13.136719 C 40.453125 11.347656 28.46875 4.847656 25.535156 3.136719 C 25.222656 2.957031 24.839844 2.957031 24.527344 3.136719 C 21.128906 5.117188 10.089844 11.621094 7.496094 13.136719 C 7.1875 13.316406 7 13.644531 7 14 L 7 20 C 7 20.378906 7.214844 20.722656 7.550781 20.894531 C 7.660156 20.949219 18.597656 26.453125 24.5 29.867188 C 24.8125 30.046875 25.195313 30.046875 25.507813 29.863281 C 27.269531 28.828125 29.117188 27.859375 30.902344 26.921875 C 32.253906 26.214844 33.636719 25.488281 35.003906 24.722656 C 35.007813 26.820313 35.003906 29.296875 35 30.40625 L 25 35.859375 L 14.480469 30.121094 C 14.144531 29.9375 13.730469 29.964844 13.417969 30.1875 L 6.417969 35.1875 C 6.140625 35.386719 5.980469 35.714844 6.003906 36.054688 C 6.023438 36.398438 6.214844 36.707031 6.515625 36.871094 L 24.542969 46.871094 C 24.695313 46.957031 24.859375 47 25.027344 47 C 25.195313 47 25.363281 46.957031 25.515625 46.875 L 43.484375 36.875 C 43.804688 36.695313 44 36.363281 44 36 C 44 36 43.992188 21.011719 44 14 Z M 25 20 L 18 16 L 25 12 L 32 16 Z\" />\n</symbol>\n</svg>"
```

Comment:
Rename to "icon_svg"

Reply:
And the '\n' should be removed, shouldn't it?

Reply:
> Rename to "icon_svg"

And the '\n' should be removed, shouldn't it?